x/swaprouter/keeper_test.go

```diff
@@ -31,10 +31,25 @@ func (suite *KeeperTestSuite) SetupTest() {
 	suite.Setup()
 }
 
-// CreateBalancerPoolsFromCoins creates balancer pools from given sets of coins.
+// createPoolFromType creates a basic pool of the given type for testing.
+func (suite *KeeperTestSuite) createPoolFromType(poolType types.PoolType) {
+	switch poolType {
+	case types.Balancer:
+		suite.PrepareBalancerPool()
+		return
+	case types.StableSwap:
+		// TODO
+		return
+	case types.Concentrated:
+		// TODO
+		return
+	}
+}
+
+// createBalancerPoolsFromCoins creates balancer pools from given sets of coins.
 // Where element 1 of the input corresponds to the first pool created,
 // element 2 to the second pool created, up until the last element.
-func (suite *KeeperTestSuite) CreateBalancerPoolsFromCoins(poolCoins []sdk.Coins) {
+func (suite *KeeperTestSuite) createBalancerPoolsFromCoins(poolCoins []sdk.Coins) {
```

Comment:
Why make testing into private method?

Reply:
It is meant to be used only inside swaprouter tests and  cannot be used anywhere else. If we want this globally, we would have to define it in  `apptesting`